### ISSUE TITLE
Improve accessibility/discoverability in Talon's default theme

### ISSUE THREAD

**Issue by tmbb** [ex-admin/ex_admin#53]
Talon's default theme has switched from using action names ("Delete", "View", "Edit") to using icons.
Icons take up less space, but text is superior for discoverability and accessibility for visually impaired users.

I suggest adding a tooltip to action buttons and maybe add something for visually impaired users (ARIA attributes? I don't know much about accessibility or screen readers). This is of course not a priority right now, but something that should be solved in the future when things are more stable.

**Comment by dsnider** [comment]
I agree about adding tooltips to the action buttons. I often prefer just links for resource actions, as done in ExAdmin, so we'll add some options here. 

What's nice with Talon is you can change the action-related markup of the `table_body.html.slim` component within your project to something like below and it's used everywhere. In this case, I even fixed a current issue (#54) where the action links/buttons wrap by adding `text-nowrap`.

```slim
    td.text-right.index-actions.text-nowrap
      a.action-show href="#{Talon.Utils.talon_resource_path(resource, :show)}" Show
      a.action-edit href="#{Talon.Utils.talon_resource_path(resource, :edit)}" Edit
      a.delete-link href="#{Talon.Utils.talon_resource_path(resource, :delete)}" Delete
```

**Comment by tmbb** [comment]
> I often prefer just links for resource actions, as done in ExAdmin

Me too, but icons do save space. I'm on the fence about this one.

> so we'll add some options here.

Given that's so easy to customize it yourself, I don't think adding options here would be necessary.
Maybe providing a different thee would be better. It would be cool if there was a mix task to install themes from a repo (github repo or something else). I don't think it's necessary to bundle all of that in talon's default installation.

**Comment by smpallen99** [comment]
I don't think we should add options. I would like to proceed with icons and add aria labels for the text. Users are free to customize the generators if they want another option. I'm also ok with adding tool tips.
